layers/+tools/dap/packages.el

```diff
@@ -137,6 +136,6 @@
           ;; Set bindings
           (apply #'spacemacs/set-leader-keys-for-major-mode mode bindings))))))
 
+(defun dap/pre-init-posframe ())
 
-(defun dap/init-posframe ()
-  (use-package posframe))
+(defun dap/post-init-posframe ())
```

Comment:
Is there a need for the `post-init`, there wasn't any special configuration for the `dap` layer before this so correct me if I'm wrong but I think just having `pre-init` should be good right?

Reply:
I think you re right. Perhaps there is no need to preserve `pre-init` also? I leave these two functions here just for not surprising the maintainer of `dap` layer. I'm okay to remove `post-init`.

Reply:
That is true, but I think it'll be better to wait for more responses too.